let fail = (msg) => {
  throw new Error(msg);
};

export {
  fail
}